### ISSUE TITLE
Ignore byproducts

### ISSUE THREAD

**Issue by werkrat** [ClaudeMetz/FactoryPlanner#304]
### Suggestion

Would it be possible to add an option to ignore byproducts when calculating production? I can't build 1.3 assembling machines so I'll usually round up (by setting a manual limit) and increase everything down the chain to maintain production, but in bigger factories I have to keep track of byproducts manually since they're included in the calculations and sometimes it's tedious.

**Comment by werkrat** [comment]
![cap6_mqIySk5](https://github.com/ClaudeMetz/FactoryPlanner/assets/17667159/c3608d43-90f8-4024-94fb-8c5a52cf3314)
![cap7_Hs162e9](https://github.com/ClaudeMetz/FactoryPlanner/assets/17667159/eb0a6ab2-2516-4651-8ab1-ea0329148563)

I added 2 screenshots of a simple factory example in case what I was asking for wasn't obvious. I set the limit for the first assembly machine high enough to produce all the copper cable for the entire factory, so the requirements are already met for the 2nd copper cable assembly machines so they're automatically set to 0. I know more subfloors will allow me to do what I want, but then I either have to fold out all subfloors or click a bunch of extra buttons to navigate my factories.
